docs/theme/components/collocator/components/switch-with-other.tsx

```
@@ -0,0 +1,71 @@
+  const { item, config, sign, parent, setSign, setConfig } = props;
+
+  const [value, setValue] = useState<boolean>(item.defaultValue ? true : false);
+
+  const changeFn = (v: boolean) => {
+    if (v) {
+      setSign(false);
+
+      if (parent) {
+        const tempConfig = item.mergeRelated && isObject(item.mergeRelated) ? Object.keys(item.mergeRelated).reduce((pre, cur) => (
+          {
+            ...pre,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)